flink-java/src/test/java/org/apache/flink/api/java/io/TextInputFormatTest.java

```diff
@@ -44,14 +48,34 @@
  * Tests for {@link TextInputFormat}.
  */
 public class TextInputFormatTest {
+
+	private File baseDir;
+	private final String tempDirName = "tempDir";
+
+	@Before
+	public void setUp() throws IOException {
```

Comment:
use a `TemporaryFolder` instead

Reply:
Seems better, I will change it then. But a little question, why my approach can not ensure the file cleaned (I check and clean the related file before execute and after test method), Thanks.

Reply:
if the JVM crashes before the delete calls are executed the files will remain.